codespell_lib/data/dictionary.txt

```diff
@@ -6976,7 +6978,9 @@ copstruction->construction
 copuright->copyright
 copurighted->copyrighted
 copurights->copyrights
+copute->compute
 coputer->computer
+coputes->computes
```

Comment:
Do you want to add the -ed version of this typo too.

Reply:
3055a10